tests/governance_history.py

```diff
@@ -96,7 +96,15 @@ def check_operations(ledger, operations):
     assert operations == set(), operations
 
 
-def check_all_tables_are_documented(ledger, doc_path):
+def get_all_tables_from_ledger(ledger):
+    table_names_in_ledger = set()
+    for chunk in ledger:
+        for tr in chunk:
+            table_names_in_ledger.update(tr.get_public_domain().get_tables().keys())
```

Comment:
Not that it matters much but there's an existing `get_latest_public_state()` function in the `Ledger` class.

Reply:
Ta, using that now.